config/routes.rb

```diff
@@ -156,6 +156,8 @@
 
           resources :site_visits, except: %i[destroy]
 
+          resources :meetings, except: %i[destroy]
```

Comment:
We don't have an edit/update yet so should be explicit here
`resources :meetings, except: %i[edit update destroy]`